aequilibrae/matrix/aequilibrae_matrix.py

```diff
@@ -254,7 +269,9 @@ def robust_name(input_name: str, max_length: int, forbiden_names: List[str]) ->
             do_cores = [x for x in cores if x in avail_cores]
             if cores != do_cores:
                 do_cores = [x for x in cores if x not in avail_cores]
```

Comment:
isn't this "missing_cores" rather than "do_cores"

Reply:
Nope.It is correct. We fix the names if needed